package.json

```diff
@@ -94,7 +94,7 @@
     "reflect-metadata": "^0.1.13",
     "regenerator-runtime": "^0.13.8",
     "sanitize-html": "^2.7.0",
-    "sequelize": "^6.6.5",
+    "sequelize": "^6.21.2",
```

Comment:
i see that there's a [breaking change](https://github.com/sequelize/sequelize/issues/14519) related to the sequelize injection vulnerability. give me a minute to quickly check if those changes affect our codebase as well

Reply:
okay looks like we dont use `:` replacements in our codebase, so we should be fine